app/middlewares/auth.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+const logger = require('../logger');
+const config = require('../../config').common.jwt;
+const { jwtMachError } = require('../errors');
+const ErrorMessages = require('../../config/error');
+
+exports.authValidator = async (req, res, next) => {
+  let token = '';
+  if (!req.headers.authorization) {
+    logger.error('The user not have authorization');
+    return next(jwtMachError(ErrorMessages.JWT_MACH_ERROR));
+  }
+  token = req.headers.authorization.substring(7);
```

Comment:
It seems to me that it is much better to extract the token with a regex, what do you think?